### ISSUE TITLE
Add subject to email when notifying the admin

### ISSUE THREAD

**Issue by PanCakeConnaisseur** [isso-comments/isso#652]
When sending an e-mail to the admin to notify him of new comments isso does not add a subject, the e-mail header is empty
`Subject: `

Please either add a static message like "New isso comment on $site" or add an option to `[smtp]` to set the subject.

isso 0.12.2-2 on Debian 10

**Comment by AlphaJack** [comment]
I am getting a subject, but it is unnecessarily encoded in quoted-printable:

```yaml
Return-Path: <isso@example.org>
Delivered-To: user@example.org
Received: from example.org
	by Example with LMTP
	id Mt3lDhhPPWBrQgAAUprYAg
	(envelope-from <isso@example.org>)
	for <user@example.org>; Mon, 01 Mar 2021 21:31:20 +0100
X-Original-To: <newcomments@example.org>
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=example.org;
	s=r; t=1614630679;
	h=from:from:reply-to:subject:subject:date:date:message-id:message-id:
	 to:to:cc:mime-version:mime-version:content-type:content-type:
	 content-transfer-encoding:content-transfer-encoding;
	bh=yXl2XThtgn54U0+CSJVvIvsKrNQqD1yvHtVu6rN8zk4=;
	b=SbSo1uEpEznji2rGECC28m4gfSlpEUA6NgOfCCKafXvSByw/lwcm869SmH8ipvcrn9kJl6
	sHW94M0PHgxr/kmiAy9N9O+wRKhrDbfIAENom1J+wzBVNKZmLQPOJRfP1gwlOJ0/3WSKYA
	xS0WFdEBsuxL6+35IfmB/ti0eblKHjjRxWti1/M5D9WtFLGDCF4xgrA/noiBrgSlBck0K5
	htATYtoqQruWmsDtdncRUWb3I6TvFiTawwSyi3MCTNZ+qe+itWE6VS1w8OO8//9NHVMg74
	FWsHBCAxYEAIbAQdLYhyubhoRp7YnmcoT2FPn5Urit3hkI1+FTV+nex5EO1hrA==
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: base64
From: "Comments | Example" <isso@example.org>
To: newcomments@example.org
Date: Mon, 01 Mar 2021 21:31:19 +0100
Subject: =?utf-8?q?New_comment_posted_on_PAGETITLE?=
Message-Id: <20210301203119.6F42C1E042A@example.org>

QW5keSA8YW5keUBjYXBwYS50bz4gd3JvdGU6CgojIExBUkdFIENPTU1FTlQKCklQIGFkZHJlc3M6
IDE5Mi4xNjguMS4wCkxpbmsgdG8gY29tbWVudDogaHR0cHM6Ly9leGFtcGxlLm9yZy91cmxcI2lz
c28tNTAKCi0tLQpEZWxldGUgY29tbWVudDogaHR0cHM6Ly9leGFtcGxlLm9yZy9jb21tZW50cy9p
ZC81MC9kZWxldGUvTlRBLllEMVBGdy5uLWdRSXRQMEo4Q2dmc2Q5NkUyTVFZVzlqeXcK
```

**Comment by ix5** [comment]
Should be fixed once https://github.com/posativ/isso/pull/553 lands.

**Comment by ix5** [comment]
#553 is probably not landing, reopening.

**Comment by pkvach** [comment]
It looks like this issue has already been fixed with these changes: https://github.com/isso-comments/isso/commit/97dd2f8075495c9bba5bb43b7805bb4f597cdbde

**Comment by ix5** [comment]
> It looks like this issue has already been fixed with these changes: [97dd2f8](https://github.com/isso-comments/isso/commit/97dd2f8075495c9bba5bb43b7805bb4f597cdbde)

Nice! That wasn't even my intention when writing that commit, but as we say in Germany, two flies with one fly swatter ;)
